.github/workflows/ci.yml

```diff
@@ -52,6 +52,7 @@ jobs:
         run: |
           echo "//registry.npmjs.org/:_authToken=${{secrets.NPM_TOKEN}}" > ~/.npmrc
           npx lerna publish --conventional-commits --changelog-preset conventional-changelog-eslint --yes
+          npx lerna publish --canary --dist-tag pre-prod --preid pre-prod --force-publish --allow-branch master --yes
```

Comment:
This is so that we always update the `pre-prod` dist tag with master when we merge. (`pre-prod` can be ahead of master, but never behind.)

Reply:
I think this is one of those times we should have a comment in the file itself to explain this.